src/main/java/lotto/model/LottoNumberGenerator.java

```diff
@@ -23,4 +24,10 @@
         Collections.sort(lottoNumber);
         return lottoNumber;
     }
+
+    public static List<Integer> getManualNumber(String manualNumber){
+        return Arrays.asList(manualNumber.split(",")).stream()
```

Comment:
```suggestion
        return Arrays.stream(manualNumber.split(","))
```
이렇게 사용할 수 있습니다:)
그리고 ","는 상수화하는게 어떨까요?